tests/test_gas_multi_demand_source_node.py

```diff
@@ -8,11 +8,18 @@
 
 
 class TestMILPGasMultiDemandSourceNode(TestCase):
-    """Unit tests for the MILP test case of 2 demands, 2 sources and a node"""
-
     def test_multi_demand_source_node(self):
-        """Test to verify that head is equal for all ports at a node. And throughout the network
-        the flow (mass) balance is maintained"""
+        """Test for case with 2 demands, 2 sources and a node. Verify that head is equal for all
```

Comment:
Suggested wording: "Test a network consisting out of 2 demands, 2 sources and a node. Verify that the head is equal for all ports at a node, and throughout the network that the flow (mass) balance is maintained

Reply:
done